dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/TableGenerator.scala

```diff
@@ -85,20 +84,21 @@ case class TableGenerator(
       val cmd = s"./dsdgen" +
         s" -TABLE $name" +
         s" -SCALE $scaleFactor" +
-        s" $parallel" +
+        s" -PARALLEL $parallelism" +
+        s" -child $i" +
         s" -DISTRIBUTIONS tpcds.idx" +
         s" -FORCE Y" +
         s" -QUIET Y"
 
       val builder = new ProcessBuilder(cmd.split(" "): _*)
       builder.directory(tempDir)
-      logger.info(s"Start $cmd at " + builder.directory())
       builder.redirectError(Redirect.INHERIT)
-      val data = Paths.get(tempDir.toString, s"${name}_${i}_$parallelism.dat")
+      logger.info(s"Start $cmd at " + builder.directory())
```

Comment:
Can we use single style? i.e. replace `+` by `${}`

Reply:
ok, i will fix it